tensorflow/core/kernels/nccl_ops.cc

```diff
@@ -13,11 +13,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-#if GOOGLE_CUDA
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 
 #include <vector>
 
+#if GOOGLE_CUDA
 #include "third_party/nccl/nccl.h"
+#endif
+#if TENSORFLOW_USE_ROCM
+#include "rccl.h"
```

Comment:
@jeffdaily could you explain a bit why `"rccl.h"` is suffice? even in StreamExecutor we need to include something like `#include "rocm/include/miopen/miopen.h"`

Reply:
I have updated the `#include` to be the full path as you indicated.  It was likely finding rccl.h in /opt/rocm/include rather than the staging location that bazel creates for the rocm platform files.